frontend/kitchen-served-fe-app/src/Home.js

```diff
@@ -79,12 +79,12 @@ const Home = () => {
     try {
       const response = await axios.post(
         `${process.env.REACT_APP_API_BASE_URL}/recipes`,
-          formData,
-          {
-            headers: {
-              'Content-Type': 'multipart/form-data',
```

Comment:
we should not be having formatting changes in pull request anymore. did you run the linting tool to format all your code before commit? if not, please run it

Reply:
I did because the js file, I correct them in vs code